tests/PowerShellPublishTests/MockIRegistrationOwnership.cs

```diff
@@ -5,6 +5,7 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using NuGet.Services.Metadata.Catalog.Ownership;
 using NuGet.Services.Publish;
 
 namespace PowerShellPublishTests
```

Comment:
We can also drop the PowerShell specific code too. The PowerShell packages are just regular NuGet packages with a different type to filter on in search.